src/assets/parsley (1).svg

```diff
@@ -0,0 +1,155 @@
+<?xml version="1.0" encoding="iso-8859-1"?>

```

Comment:
filename suggests that its a copy-paste error ;) 
Maybe it should be renamed?

Reply:
i've chosen other parsley, soo this one i just deleted.